src/solc_0.8/raffle/GenericRaffle.sol

```diff
@@ -251,10 +275,13 @@ contract GenericRaffle is ERC721EnumerableUpgradeable, OwnableUpgradeable, Reent
     }
 
     function setAllowedExecuteMint(address _address) external onlyOwner {
+        require(_address != address(0x0), "Address is zero address");
         allowedToExecuteMint = _address;
+        emit AllowedExecuteMintSet(_address);
     }
 
     function setSandOwnerAddress(address _owner) external onlyOwner {
+        require(_owner != address(0x0), "Owner is zero address");
         sandOwner = _owner;
```

Comment:
Missing emit event

Reply:
Does it really make sense to emit an event for setters such as `setSandOwnerAddress`? I can do it, but I thought it doesn’t make much sense, which is why I didn't add it for these.